mailet/icalendar/src/main/java/org/apache/james/transport/mailets/ICalendarParser.java

```diff
@@ -142,13 +142,17 @@ private Stream<Pair<String, Calendar>> createCalendar(String key, byte[] icsCont
             ByteArrayInputStream inputStream = new ByteArrayInputStream(icsContent);
             return Stream.of(Pair.of(key, builder.build(inputStream)));
         } catch (IOException e) {
-            if (LOGGER.isErrorEnabled()) {
-                LOGGER.error("Error while reading input: " + new String(icsContent, StandardCharsets.UTF_8), e);
+            if (LOGGER.isDebugEnabled()) {
+                LOGGER.debug("Error while reading calendar input: " + new String(icsContent, StandardCharsets.UTF_8), e);
+            } else {
+                LOGGER.info("Error while reading calendar input", e);
```

Comment:
Does that represent really that many logs? 


I find weird TBH having a log message starting by `Error...` and not being on error level but info instead?

Reply:
Well ERROR logs means "wo po po something CRITICAL happened and you as an admin need to get an urgent look"

Benign failed parsing clearly do not qualify as such.